### PR TITLE
[zh-cn]: create the translation of Document `append()` method

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/document/append/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Document：append() 方法
+slug: Web/API/Document/append
+l10n:
+  sourceCommit: 542ef6cfd82288925e0a9238b47933f03e2dddca
+---
+
+{{APIRef("DOM")}}
+
+**`Document.append()`** 方法会在文档的最后一个子节点后插入一组{{domxref("Node", "节点", "1", "nocode")}}对象或字符串对象。字符串对象被插入为相同{{domxref("Text", "文本", "1", "nocode")}}节点。
```
Comment (T34-active) [inline files/zh-cn/web/api/document/append/index.md:10]:
```suggestion
**`Document.append()`** 方法会在文档的最后一个子节点后插入一组{{domxref("Node", "节点", "", 1)}}对象或字符串对象。字符串对象被插入为相同{{domxref("Text", "文本", "", 1)}}节点。
```

File: files/zh-cn/web/api/document/append/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+slug: Web/API/Document/append
+l10n:
+  sourceCommit: 542ef6cfd82288925e0a9238b47933f03e2dddca
+---
+
+{{APIRef("DOM")}}
+
+**`Document.append()`** 方法会在文档的最后一个子节点后插入一组{{domxref("Node", "节点", "", 1)}}对象或字符串对象。字符串对象被插入为相同{{domxref("Text", "文本", "", 1)}}节点。
+
+此方法将子级追加到 `Document` 中。要追加到树中的任意元素，请参阅 {{domxref("Element.append()")}}。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/document/append/index.md:12]:
```suggestion
此方法将一个子元素追加到 `Document` 中。要向树中的任意元素添加子元素，请参阅 {{domxref("Element.append()")}}。
```

File: files/zh-cn/web/api/document/append/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+```js-nolint
+append(param1)
+append(param1, param2)
+append(param1, param2, /* …, */ paramN)
+```
+
+### 参数
+
+- `param1`, …, `paramN`
+  - : 要插入的 {{domxref("Node")}} 或字符串对象集。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/document/append/index.md:25]:
```suggestion
  - : 要插入的一组 {{domxref("Node")}} 或字符串对象。
```

File: files/zh-cn/web/api/document/append/index.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+如果你尝试在现有 HTML 文档中添加元素，可能会抛出 `HierarchyRequestError` {{domxref("DOMException")}}，因为 {{HTMLElement("html")}} 元素已经存在。
+
+```js
+let html = document.createElement("html");
+document.append(html);
+// HierarchyRequestError: The operation would yield an incorrect node tree.
+```
+
+如果创建的新文档没有任何现有元素，你可以附加 HTML 根元素（或 SVG 根元素）：
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/document/append/index.md:48]:
```suggestion
如果创建的新文档没有任何现有元素，你可以添加一个 HTML 根元素（或 SVG 根元素）：
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions for your reference.

**Review by JasonLamv-t — APPROVED** [review]
Sorry for the late. LGTM, thanks! @T34-active

**Pull request merged**